public_records_portal/upload_helpers.py

```diff
@@ -263,6 +263,8 @@ def allowed_file(file):
     :param file: pass in a file that will be checked for allowed mimetype
     :return: True if the mimetype is allowed or False if its not allowed
     """
+    if file.filename == u'':
```

Comment:
This doesn't make sense. If no file is passed in, and the program still gets to this point, then we should return False, not True